features/285-design-webhooks/events.md

```diff
@@ -58,16 +58,12 @@ information about the event:
 For BEFORE events about a user, following fields can be mutated:
 - `metadata`: user metadata
 - `verify_info`: verify info of user
-- `is_verified`: verified status of user
```

Comment:
Why remove this attribute and how is it related with #336 ?

Reply:
`is_verified` is a computed field. It would make no sense to modify it.

I suppose I'll allow `is_manually_verified` flag to be mutated instead.